src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+if (thumbs && largeImg) {
+  thumbs.addEventListener('click', handleThumbnailClick);
+}
+
+function handleThumbnailClick(e) {
+  e.preventDefault();
+
+  const link = e.target.closest('.list-item__link');
+
+  if (link) {
+    updateLargeImage(link.href);
```

Comment:
The `link.href` might not be a valid URL if the `link` element does not have an `href` attribute. Consider adding a check to ensure `link.href` is valid before using it.